xmrstak/cli/cli-miner.cpp

```diff
@@ -230,10 +244,30 @@ int main(int argc, char *argv[])
 		;
 		configEditor configTpl{};
 		configTpl.set(std::string(tpl));
+		std::cout<<"Please enter:"<<std::endl;
+		auto& currency = params::inst().currency;
+		if(currency.empty())
+		{
+			std::string tmp;
+#if defined(CONF_NO_AEON)
+			tmp = "xmr";
```

Comment:
Can you change xmr to proper currency name - Monero?

Reply:
Yes I can change it to 'monero'. I used 'xmr' to keep it short but you are right monero could be better.